packages/client-core/src/common/components/TouchGamepad/index.tsx

```diff
@@ -52,22 +59,32 @@ export const TouchGamepad: FunctionComponent<TouchGamepadProps> = () => {
       size: size,
       dynamicPage: true
     })
-    const targetElement = stickLeft[0].ui.el
-    targetElement.addEventListener('touchstart', (ev) => {
-      enableInput({ mouse: false })
-    })
-    targetElement.addEventListener('touchend', (ev) => {
-      enableInput({ mouse: true })
-    })
-
+ 
+	const canvasElement = (document.getElementById('engine-renderer-canvas') as HTMLCanvasElement)
     stickLeft.on('move', (e, data) => {
+	
+	  if (canvasElement.addEventListener) {  // all browsers except IE before version 9
```

Comment:
Maybe this is right, can't check it, but this seems like every time the stick moves, we are adding a new even listener?